pkg/util/dynamichelper/unstructuredobj.go

```diff
@@ -0,0 +1,60 @@
+package dynamichelper
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	kruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type UnstructuredObj struct {
+	obj unstructured.Unstructured
+}
+
+func NewUnstructuredObj(obj *unstructured.Unstructured) *UnstructuredObj {
+	return &UnstructuredObj{*obj}
+}
```

Comment:
Is this really useful? seems like it's an unnecessary wrapping

Reply:
and also, why did you change all the methods to use pointers in your last commit?

Reply:
feel better to have a construction func first.

it is because of the previous change, where the obj to be added to the []kruntime.Object changed from UnstructuredObj to *UnstructuredObj for better readability, so just let *UnstructuredObj be a kruntime.Object here.

Reply:
it isn't useful.